lib/inherit.js

```diff
@@ -106,7 +104,7 @@ function applyMixins(mixins, res) {
                 inherit(mixins[0], mixin.prototype, mixin) :
                 inherit(mixins[0], mixin);
     }
-    return res || mixins[0];
+    return res;
```

Comment:
I did not find the case that can use it.
If I'm wrong, please show me this case.